websocket_test.go

```diff
@@ -21,6 +31,124 @@ func init() {
 	RegisterNewService(serviceWebSocket, newServiceWebSocket)
 }
 
+// Adapted from 'https://golang.org/src/crypto/tls/generate_cert.go'
+func generateSelfSignedCert() (string, string, error) {
+	var (
```

Comment:
`var` is very useful outside of methods, but not needed inside. You can directly write:
```
host := "127.0.0.1"
```